Conquer/src/main/java/conquer/utils/Updater.java

```diff
@@ -0,0 +1,55 @@
+package conquer.utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.file.Files;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import conquer.data.Shared;
+import conquer.data.Version;
+
+public class Updater {
+
+    private static final Updater INSTANCE = new Updater();
+    private Version latestVersion;
+
+    public static void update(Function<Boolean, Boolean> callback, Consumer<Boolean> downloadFinished) {
+        if (callback.apply(INSTANCE.hasToUpdate())) {
+            try {
+                INSTANCE.downloadFile(new URL("https://raw.githubusercontent.com/JCWasmx86/JCWasmx86.github.io/master/conquer-data/windows.zip"), Shared.BASE_DIRECTORY + "updates" + System.getProperty("file.separator") + "data.zip");
```

Comment:
This URL is not right anymore. It should be either `data_amd64.zip` or `data_i386.zip` depending on the architecture of the PC/installation.